lib/go-tc/steeringtarget.go

```diff
@@ -47,9 +47,6 @@ type SteeringTargetNullable struct {
 
 func (st SteeringTargetNullable) Validate(tx *sql.Tx) error {
 	errs := []string{}
-	if st.TargetID == nil {
```

Comment:
Because the validation applies to the object. Not sure what you mean by "controller". If you mean the "CRUDer" objects, only a small percentage of our endpoints are "CRUD", by making all `Validate` funcs members of the real objects, we prevent them being scattered, and can keep them all in one place.

Putting it in `lib/tc` also makes the validation accessible to external libraries, if it's useful to someone to do their own validation.